pakyow-core/lib/pakyow/core/router.rb

```diff
@@ -300,7 +302,13 @@ def handle(name_exception_or_code, as: nil, &block)
     
       def exception(klass, context: nil, handlers: {}, exceptions: {})
         exceptions = self.exceptions.merge(exceptions)
-        return unless exception = exceptions[klass]
+        unless exception = exceptions[klass]
+          return unless parent
+
+          parent.exception(
```

Comment:
I don't think this is quite right, since the code starting at line 313 relies on `exception` being set. It might be more clear if we create a `Router#exception_for_class` method that searches passed exceptions, exceptions in self, and then calls `exception_for_class` on `parent` if nothing is found. Then line 303 would become:

```ruby
return unless exception = exception_for_class(klass, exceptions: exceptions)
```

We also don't have to pass `exceptions` when calling the parent. Reason is, if there was a match we wouldn't be looking at the parent to begin with. Let me know what you think and please correct me if I'm wrong here!